src/ContainerHeader.jsx

```diff
@@ -15,7 +15,7 @@ const ContainerHeader = ({ user, twoOwners, ownerFilter, handleOwnerChanged, tex
                             {_("Owner")}
                         </ToolbarItem>
                         <ToolbarItem>
-                            <FormSelect id="containers-containers-owner" value={ownerFilter} onChange={handleOwnerChanged}>
+                            <FormSelect id="containers-containers-owner" value={ownerFilter} onChange={(_, value) => handleOwnerChanged(value)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-080503-570e483c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)